src/modal/_pera-wallet-connect-modal.scss

```diff
@@ -1,73 +1,116 @@
-.pera-wallet-connect-modal__display-qr-code-button,
-.pera-wallet-connect-modal__launch-pera-wallet-button {
-  letter-spacing: -0.3px;
+@import "../ui/styles/media-queries";
+
+.pera-wallet-connect-modal__content {
+  display: grid;
+  grid-template-columns: 205px auto;
+  gap: 80px;
 }
 
-.pera-wallet-connect-modal__launch-pera-wallet-button {
+.pera-wallet-connect-modal__launch-pera-wallet-button,
+.pera-wallet-connect-modal__install-pera-wallet-button,
+.pera-wallet-connect-modal__cancel-button {
   display: block;
 
   padding: 14px;
-  margin-bottom: 20px;
-
-  border-radius: 4px;
-  background-color: var(--pera-wallet-main-color);
 
-  color: black;
+  border-radius: 12px;
 
   text-decoration: none;
   text-align: center;
   font-size: 14px;
+  line-height: 20px;
+  letter-spacing: -0.09px;
+  font-weight: 500;
 }
 
-.pera-wallet-connect-modal__cancel-button,
-.pera-wallet-connect-modal__display-qr-code-button {
+.pera-wallet-connect-modal__launch-pera-wallet-button {
+  margin-bottom: 32px;
+
+  background-color: #6b46fe;
+
+  color: #ffffff;
+}
+
+.pera-wallet-connect-modal__install-pera-wallet-button {
+  margin-bottom: 20px;
+
+  background-color: #f2f3f8;
+
+  color: #2c3559;
+}
+
+.pera-wallet-connect-modal__cancel-button {
   width: 100%;
 
-  padding: 14px;
-  margin: 0;
+  background-color: #f2f3f8;
 
-  background-color: white;
-  border: 1px solid black;
-  border-radius: 4px;
+  color: #2c3559;
+}
 
-  color: black;
+.pera-wallet-connect-modal__new-to-pera-box {
+  position: relative;
+
+  margin-bottom: 32px;
+
+  border-top: 1px solid #e6e8ee;
 }
 
-.pera-wallet-connect-modal__qr-code {
-  cursor: none;
+.pera-wallet-connect-modal__new-to-pera-box__text {
+  position: absolute;
+  top: -12px;
+  right: calc(50% - 56px);
```

Comment:
can we do it with flexbox rather than absolute positioning?
 
Could you please help me to understand right positioning here?
if 56px is the half-width of the container, could we use? 
```transform: translateX(-50%); ```

Reply:
I used absolute positioning for this. I created some div and add border-top to this div. I'm not sure how we can do it with flexbox.
<img width="675" alt="Screen Shot 2022-06-09 at 10 23 47" src="https://user-images.githubusercontent.com/54077855/172789227-59ccdacb-2143-4281-ba68-9c223a4ba1a2.png">